geomesa-tutorials-kafka/geomesa-tutorials-kafka-streams-quickstart/README.md

```diff
@@ -0,0 +1,4 @@
+GeoMesa Kafka Streams Quick-Start Tutorial
+==================================
+
+See the official GeoMesa [documentation](http://www.geomesa.org/documentation/tutorials/geomesa-quickstart-kafka-streams.html) for instructions.
```

Comment:
Is there an associated PR for the docs change with this PR?

Reply:
https://github.com/locationtech/geomesa/pull/2903